test/server/config_validation/xds_fuzz.cc

```diff
@@ -0,0 +1,235 @@
+#include "test/server/config_validation/xds_fuzz.h"
+
+#include <fstream>
+
+#include "envoy/event/dispatcher.h"
+
+#include "test/common/grpc/grpc_client_integration.h"
+#include "test/integration/fake_upstream.h"
+
+namespace Envoy {
+namespace Server {
+
+const std::string config = R"EOF(
```

Comment:
Can we use https://github.com/envoyproxy/envoy/blob/master/test/config/utility.h to avoid having custom bootstrap?  I see that we do what you have here in `ads_integration_test`, so probably fine, but I was never a big fan of this.